process/consensus.md

```diff
@@ -9,24 +9,96 @@ Consensus is critical in a standards process:
+sufficient:
+
+* Contributors of *different affiliation* are individuals who are affiliated
+  with different companies or institutions.
+* *Interested parties* are individuals, companies, or institutions who usually
+  participate in the WebAssembly Community Group or Working Group and are
+  interested in a particular topic.
+* *Small group* is a subset of Community Group and Working Group participants
+  who decide to collaborate on a single targeted proposal.
+
+We differentiate the following cases:
+
+1. Editors are empowered to make changes that have no technical impact without
+   explicit group decision. GitHub issues and pull requests are sufficient.
+   Their title should identify them as editorial. Closing or merging them after
+   a full work day of positive review by at least one contributor of different
+   affiliation is acceptable as long as there is no objection. Editors shall
+   revisit the editorial nature of the change if there is any contention.
+
+2. Small technical changes or additions can be discussed in GitHub issues and
+   pull requests. It is the author's responsibility to ensure that interested
+   parties sign off. Simply mentioning people on GitHub may go unnoticed, the
+   onus remains on the author to contact interested parties and obtain their
+   feedback. Closing or merging after a full week of positive review by at least
+   three contributors of different affiliations is acceptable as long as there
+   is no objection. Consensus will be deemed to not have been reached if
+   interested parties did not sign off. At any point in time a contributor can
+   request that final consensus be delayed to a CG meeting. In this case, the
+   chair puts the item on the group's agenda of upcoming discussions.
+
+3. Substantial technical changes or additions are developed in their own GitHub
+   repository by a champion. It is critical that these proposals be able to
+   evolve quickly without much process. Early on in such a proposal's lifetime
+   no consensus is needed and a single champion can modify the proposal at will.
+   As the proposal matures it is expected that the champion will seek
+   collaborators to form a small group or
+   [subgroup](https://github.com/WebAssembly/meetings/blob/main/process/subgroups.md).
+   Gauging consensus within small groups is left up to the champion and gauging
+   consensus within subgroups is left up to the subgroup chair.
+
+Only 1. and 2. apply to the Working Group since the Community Group is the sole
+venue where substantial work can occur. It is expected that Working Group
+changes or additions in 2.'s category are validated by the Community Group
+before being adopted by the Working Group. It is expected that all changes or
+additions which reach consensus in the Community Group will be forwarded to the
+Working Group for adoption.
+
+When a proposal is near maturity the champion shall bring it to a CG or subgroup
```

Comment:
this phrasing ("maturity") seems a little ambiguous, since what "maturity" means is fairly well-defined by the phase process and it's really more of a continuum than a binary thing.

actually, the next paragraph refers to the phase process specifically. maybe they could be combined, to say that the champion brings a proposal up for formal consensus for phase advancement (which is basically a recognized increase in maturity level), and also that champions (or really any interested party) can bring open design questions and contentious issues to the CG or subgroup.

Reply:
does the thumbs up mean you were planning on changing this? or you think it's best as is?

Reply:
Not sure what happened here. I thought I had made the change but it looks like I just... didn't??? Will fix.

Reply:
Ok, looks like I just forgot to delete the first paragraph after updating the second. It should be fixed now.